curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md

```diff
@@ -120,9 +119,10 @@ assert.equal(document.querySelectorAll('h2')[0]?.innerText, 'Packages');
 You should have a `p` element below your first `h2` element.
 
 ```js
+const h2 = document.querySelector('h2');
+assert.exists(h2);
```

Comment:
With these changes you don't need these two lines anymore
```suggestion
```

Reply:
You're right. I've removed them.